deploy.desc

```diff
@@ -7,4 +7,4 @@ SECRETS=(SPLITIO_API_KEY SPLITIO_EXT_API_KEY)
 CPU_REQUEST="5m"
 MEMORY_REQUEST="85Mi"
 NGINX_CPU_REQUEST="15m"
```

Comment:
Should I also increase the NGINX_CPU_REQUEST?

Reply:
yea, I would bump it up to 50m,  these small number are so tiny I have trouble believing them.